pkg/codegen/schema.go

```diff
@@ -733,7 +733,7 @@ func GenFieldsFromProperties(props []Property) []string {
 				fieldTags["yaml"] = p.JsonFieldName + ",omitempty"
 			}
 			if p.NeedsFormTag {
-				fieldTags["form"] = p.JsonFieldName + ",omitempty"
+				fieldTags["form"] = p.JsonFieldName
```

Comment:
This got my test passing, but I wasn't sure if it was a good idea. If it is a good idea, does it merit a backwards compatibility flag?

Summarising https://github.com/oapi-codegen/oapi-codegen/issues/1262#issuecomment-2868180395, echo doesn't consider `omitempty` to make sense in forms. Not sure about other HTTP servers. I quick search found https://github.com/go-playground/form/issues/18 but I don't think that's used in an HTTP server like echo or gin etc.